e2e/stack.go

```diff
@@ -41,16 +43,19 @@ type EventListener interface {
 }
 
 type StackConfig struct {
-	L1URL *url.URL
-	*rollup.Config
-	Operator L1User
-	Users    []L1User
+	L1URL         *e2eurl.URL
+	RUConfig      *rollup.Config
+	Operator      L1User
+	Users         []L1User
+	L1Client      *L1Client
+	L2Client      *bftclient.HTTP
+	MonomerClient *MonomerClient
 }
 
-type Stack struct {
-	monomerEngineURL *url.URL
-	monomerCometURL  *url.URL
-	opNodeURL        *url.URL
+type stack struct {
```

Comment:
Do you think it would be better to remove this struct? It doesn't serve any purpose.

Reply:
It would probably be better to remove, but it currently still shepherds some data into the constructor for `e2e.OPStack`, and make for a bigger change set than necessary / convenient for the immediate purpose of cleaning up the e2e tests.

Reply:
sgtm, better for follow-up then